Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/MenuAccessibilityExtensions.kt

```diff
@@ -18,3 +22,11 @@ fun Menu.setItemContentDescription(context: Context) {
         }
     }
 }
+
+fun Menu.setTextSize() {
```

Comment:
can you please a common style for the whole App, See `Theming menus` under
https://m3.material.io/components/menus/implementation

Reply:
Thanks, I'll have a look. I did try to go this route first, but the answers I found were saying to include this line in my theme `app:itemTextAppearance="@style/menu_text_style"` but it didn't have any effect.